From f3e38e43a72af31abf9ce7bdb614eae1c47249a4 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Thu, 23 Dec 2021 23:06:50 -0700
Subject: [PATCH] introduce native source packages slightly differently

Taking the definitions section in ch. 1 strictly, "upstream releases" refers
to the content of upstream's releases, not the events of upstream releasing.
So when we say that a native source package does not distinguish between
Debian packaging and upstream releases, we mean that there is no distinction
made between the debian/ directory and all the rest of the source code.

This is true if we are thinking about our source package formats, where for a
non-native package there are separate tarballs for debian/ and for the rest of
the source code.  However, the distinction between native and non-native
packages is not inexorably tied to our use of source packages.  If someone
doesn't have source package formats in mind, the distinction between the
debian/ directory and all the rest of the source code is not so salient -- in
a sense, there is always *some* distinction between those, native or not.
---
 policy/ch-source.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy/ch-source.rst b/policy/ch-source.rst
index d6fbfc7..65e6f9e 100644
--- a/policy/ch-source.rst
+++ b/policy/ch-source.rst
@@ -14,7 +14,7 @@ the ``debian`` directory of the unpacked source package.
 Debian source packages are classified as *native* or *non-native*.
 
 A native source package is one that does not distinguish between Debian
-packaging and upstream releases.  A native source package contains a
+packaging releases and upstream releases.  A native source package contains a
 single tar file of source material, and the versioning does not have a
 Debian-specific component.  Native packages are normally (but not
 exclusively) used for software that has no independent existence outside
-- 
2.30.2

